src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -66,7 +66,10 @@ export const InputBar = ({
         }
         plain
         size={isChatInput ? '16px' : '13px'}
-        style={{ fontWeight: isChatInput ? 'normal' : 'bold' }}
+        style={{
+          fontWeight: isChatInput ? 'normal' : 'bold',
```

Comment:
I think you should remove `isChatInput` and parent component should configure directly fontWeight

Reply:
I would prefer not to, because IMO if I pass fontWeight value in props, I should do the same for all other conditional css properties in the component.